runners/apex/pom.xml

```diff
@@ -187,6 +187,7 @@
               <groups>org.apache.beam.sdk.testing.RunnableOnService</groups>
               <excludedGroups>
                 org.apache.beam.sdk.testing.UsesStatefulParDo,
+                org.apache.beam.sdk.testing.UsesTimersInParDo,
```

Comment:
I agree with this, for discoverability. We should switch them both to be nouns, and some naming scheme like `UsesXyzInParDo` since not all features can be phrased as an adjective like "stateful". Or potentially namespace them like `Feature.ParDo.State`.